odl/contrib/fom/test/test_supervised.py

```diff
@@ -68,5 +68,62 @@ def test_psnr(space):
     assert result == pytest.approx(expected)
 
 
+def test_mean_value_difference_sign():
+    space = odl.uniform_discr(0, 1, 10)
+    I0 = space.one()
+    I1 = -I0.copy()
+    assert np.abs(odl.contrib.fom.mean_value_difference(I0, I1)) > 0.1
+    assert np.abs(odl.contrib.fom.mean_value_difference(I0, I1,
+                  normalized=True)) > 0.1
+
+
+def test_mean_value_difference_range_value(space):
+    I0 = space.element(np.random.normal(0, 1, size=space.shape))
```

Comment:
there is a utility `odl.util.testutils.noise_element` which does exactly this (but works with e.g. complex numbers etc equally well).